packages/ui-components/src/components/radio-list-item/radio-list-item.helper.tsx

```diff
@@ -0,0 +1,29 @@
+import { JSX, h } from '@stencil/core';
+import { LINK_MATCHER_REGEX } from './radio-list-item.config';
+import { EAnchorTarget } from '../../types';
+
+export const buildDescription = (description?: string): JSX.Element[] => {
```

Comment:
Probably we can export this util to be used in other situations like this?

Reply:
As discussed earlier, this bit of code is still attached / used by this component only, moving forward, we'll have a separation of this parsing logic from the component.